docker-compose.yml

```diff
@@ -69,7 +69,7 @@ services:
     hostname: openldap
     container_name: openldap
     ports:
-        - 389:389
+        - 1389:389
```

Comment:
Can you please remind me why this port change was required?

Reply:
Maybe to avoid interfering with folks who are running ldap locally on 389? Not sure it's in the scope of this PR

Reply:
Does this port need to be exposed on the host at all?

Reply:
I changed it because I'm getting on Gitpod permissions issues and ports used < 1024 for example, see:

![CleanShot 2021-09-17 at 12 06 08](https://user-images.githubusercontent.com/4061923/135729799-97d885ff-6b5e-49a1-872e-9179750a6999.jpg)
 
But I agree with @rspurgeon, there is no need to expose it, it is only used from within containers

Should I remove it completely or keep it like that ?

Reply:
Yes, I believe we can remove this mapping entirely as long as we verify no broken behavior (of course)

Reply:
I've removed it entirely